SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -9,14 +9,12 @@
 #include "SpatialConstants.h"
 #include "SpatialGDKSettings.h"
 
-#include <WorkerSDK/improbable/c_worker.h>
```

Comment:
Why remove the include?

Reply:
Not needed by anything

Reply:
This was actually just my linter being overzealous. Will add it back in as this file does depend on it.